### DISCUSSION TITLE
Is it possible to set word wrap or automatically update the width of a table cell?

### DISCUSSION THREAD

**Discussion by samuelchristlie** [fyne-io/fyne#4912] [Q&A] [answered]
I'm working on a program that fetches data from a database and displays it onto a table. The problem is that the width of each cell is set during the creation of the cells, so long strings would overlap with the cell next to it.

One idea that I have is to change the column width after the table is created or trim the text to a certain length, but I'm wondering if there's a better way to do it.

**Comment by andydotxyz** [comment, marked as answer]
You can truncate the text to fit, or you can update the row height or column width (which you do will depend on whether you wrap content).

This is not done automatically as the table widget is optimised for fast re-use to scroll large data. What you describe is possible through a little more code in your app.
